maintainers/maintainer-list.nix

```diff
@@ -13276,6 +13276,12 @@
     githubId = 57180880;
     name = "Ansh Tyagi";
   };
+  therishidesai = {
```

Comment:
Add yourself as a maintainer in a separate commit - no necessity of a separate _pull request_.

Follow the tutorial:

https://stackoverflow.com/questions/6217156/break-a-previous-commit-into-multiple-commits